shared/services/config/stader-config.go

```diff
@@ -159,20 +161,30 @@ func LoadFromFile(path string) (*StaderConfig, error) {
 	if err := yaml.Unmarshal(configBytes, &settings); err != nil {
 		return nil, fmt.Errorf("could not parse settings file: %w", err)
 	}
+	// to be removed : start-01
```

Comment:
if you think this is unnecessary, remove it

Reply:
removing the whole printing og config object, keeping the printing of ssv-flag, will remove after successful dev testing.